webhook.rst

```diff
@@ -16,8 +20,75 @@ Installation
 
     $ composer require symfony/webhook
 
+
+Consuming Webhooks
+------------------
+
+Consider a third-party API that allows you to track stock levels for various products and can send webhooks to your
+Symfony application.
+
+From the perspective of your application (the *consumer*), which receives the webhook, three primary phases need to be anticipated:
+
+1) Receiving the webhook
+
+2) Verifying the webhook and constructing the corresponding Remote Event
+
+3) Using the received data.
+
+Symfony Webhook, when used alongside Symfony RemoteEvent, streamlines the management of these fundamental phases.
+
+A Single Entry Endpoint: Receive
+--------------------------------
+
+Through the built-in :class:`Symfony\\Component\\Webhook\\Controller\\WebhookController`, a unique entry point is offered to manage all webhooks
+that our application may receive, whether from the Twilio API, a custom API, or other sources.
+
+By default, any URL prefixed with ``/webhook`` will be routed to this :class:`Symfony\\Component\\Webhook\\Controller\\WebhookController`.
+Additionally, you have the flexibility to customize this URL prefix and rename it according to your preferences.
+
+.. code-block:: yaml
+
+    # config/routes/webhook.yaml
+    webhook:
+        resource: '@FrameworkBundle/Resources/config/routing/webhook.xml'
+        prefix: /webhook # or possible to customize
+
+Additionally, you must specify the parser service responsible for analyzing and parsing incoming webhooks.
+It's crucial to understand that the :class:`Symfony\\Component\\Webhook\\Controller\\WebhookController` itself remains provider-agnostic, utilizing
+a routing mechanism to determine which parser should handle incoming webhooks for analysis.
+
+As mentioned earlier, incoming webhooks require a specific prefix to be directed to the :class:`Symfony\\Component\\Webhook\\Controller\\WebhookController`.
+This prefix forms the initial part of the URL following the domain name.
+The subsequent part of the URL, following this prefix, should correspond to the routing name chosen in your configuration.
+
+The routing name must be unique as this is what connects the provider with your
+webhook consumer code.
+
+.. code-block:: yaml
```

Comment:
this should use a `configuration-block` to show the configuration in all formats